core/utils/device/device.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, id, name, type, version):
             type = type.replace(',', '')
             type = type.replace('ProductType:', '').strip(' ')
             self.name = type
+            self.device_log = SimAuto.get_logs(self.id)
```

Comment:
Can we rename it to make it clear it is path.

Reply:
done